clmm/clusterensemble.py

```diff
@@ -0,0 +1,235 @@
+"""@file clusterensemble.py
+The Cluster Ensemble class
+"""
+import numpy as np
+from scipy.stats import binned_statistic
+from astropy.table import Table
+from .plotting import mpl
+import healpy
+
+from .gcdata import GCData
+from .galaxycluster import GalaxyCluster
+from .utils import compute_radial_averages, convert_units
+from .dataops import make_radial_profile, make_stacked_radial_profile
+
+class ClusterEnsemble():
+    """Object that contains a list of GalaxyCluster objects
+
+    Attributes
+    ----------
+    unique_id : int or string
+        Unique identifier of the galaxy cluster ensemble
+    data : GCData
+        Table with galaxy clusters data (i. e. ids, profiles, redshifts).
+    id_dict: dict
+        Dictionary of indicies given the cluster id
+    """
+    def __init__(self, unique_id, gclist, *args, **kwargs):
```

Comment:
It would be more clear by mentioning `**kwargs` are eventually passed to `make_individual_radial_profile`.

Reply:
Hmm, not sure - I'll defer to @m-aguena and @payerne who wrote that module.